### ISSUE TITLE
Add `@capture`, which captures both `err` & `out`?

### ISSUE THREAD

**Issue by singularitti** [JuliaIO/Suppressor.jl#35]
Like the title, a counterpart of `@suppress`, which covers `@suppress_err` & `@suppress_out`.

**Comment by j-fu** [comment]
Yeah would be useful [here]( https://github.com/fonsp/PlutoUI.jl/pull/31). Could they be returned in a tuple of two strings?

**Comment by j-fu** [comment]
See https://github.com/JuliaIO/Suppressor.jl/pull/36
